pkg/cmd/server/api/validation/oauth.go

```diff
@@ -134,6 +134,10 @@ func ValidateIdentityProvider(identityProvider api.IdentityProvider) ValidationR
 		case (*api.LDAPPasswordIdentityProvider):
 			validationResults.Append(ValidateLDAPIdentityProvider(provider))
 
+		case (*api.KeystonePasswordIdentityProvider):
+			validationResults.AddErrors(ValidateRemoteConnectionInfo(provider.RemoteConnectionInfo).Prefix("provider")...)
```

Comment:
move the ValidateRemoteConnectionInfo call into ValidateKeystoneIdentityProvider